pandas/tseries/resample.py

```diff
@@ -1204,6 +1206,13 @@ def _get_time_delta_bins(self, ax):
         end_stamps = labels + 1
         bins = ax.searchsorted(end_stamps, side='left')
 
+        if ax.hasnans:
```

Comment:
NaT group will be ignored in aggregation. It is handled the same way as in function `_get_time_bins`.